src/pages/terms-and-conditions.js

```diff
@@ -1,85 +1,97 @@
-import { useEffect, useState } from "react"
-import { useRouter } from "next/router"
+import { useEffect, useState } from "react";
+import { useRouter } from "next/router";
 import { getFirestore, collection, getDocs } from "firebase/firestore";
-import styles from "@/styles/termsAndConditions.module.scss"
-import IndexLayout from "@/Layouts/IndexLayout"
-import Link from "next/link"
-import lt from '@/locales/lt'
-import en from '@/locales/en'
+import styles from "@/styles/termsAndConditions.module.scss";
+import IndexLayout from "@/Layouts/IndexLayout";
+import Link from "next/link";
+import lt from "@/locales/lt";
+import en from "@/locales/en";
 
 export default function TemrsAndConditions() {
-
-   const db = getFirestore();
-   const colRef = collection(db, "ltl-test")
-   getDocs(colRef)
-   .then((snapshot) => {
-      let tncList = []
+  const db = getFirestore();
+  const colRef = collection(db, "ltl-test");
+  getDocs(colRef)
+    .then((snapshot) => {
+      let tncList = [];
       snapshot.docs.forEach((tab) => {
-         tncList.push({ ...tab.data(), id: tab.id })
-      })
-      console.log(tncList)
-   })
-   .catch((err) => {
-      console.log(err.message)
-   })
+        tncList.push({ ...tab.data(), id: tab.id });
+      });
+      console.log(tncList);
```

Comment:
Can we delete this. ?

Reply:
no, its for future firebase storage data colecting code example, it shud stay until we finish with firebase